api/models/location.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const locationSchema = new Schema({
```

Comment:
What is this being used for? What does `name` represent?

Reply:
This is the mongodb schema setup(using mongoose pkg), figured name could be a city, county, geographical region, etc.

Reply:
This is 100% my own ignorance of Mongo, so sorry for hammering you with questions, just trying to understand more of what’s happening. How is this useful? What will be stored here and how will we access it?

Reply:
Your good, sorry I know I should comment more. This schema with mongoose sets up the fields for the documents we're trying to save to db and at the same time creates the class with methods we can use to do things like save, find, findByID, delete all real easy with this schema. some basic examples in the server.js